spec.md

```diff
@@ -1257,6 +1257,9 @@ This RPC is typically called by the CO when the workload using the volume is bei
 
 This operation MUST be idempotent.
 If the volume corresponding to the `volume_id` is not attached to the node corresponding to the `node_id`, the Plugin MUST reply `0 OK`.
+If the volume corresponding to `volume_id` or the node corresponding to
+`node_id` cannot be found by the Plugin and the volume can be safely regarded as
+ControllerUnpublished from the node, the plugin MAY return `0 OK`.
```

Comment:
Does the `ControllerUnpublishVolume Errors` section below need to be updated at all?

Reply:
added a conditional to the two `NotFound` errors